docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -428,80 +429,80 @@ Events of pull requests.
-| `updated_at`       | datetime        | 3         |                                                                 |         |
-| `name`             | varchar         | 255       |                                                                 |         |
-| `commit_sha`       | varchar         | 255       |                                                                 |         |
-| `branch`           | varchar         | 255       |                                                                 |         |
-| `repo`             | varchar         | 255       |                                                                 |         |
-| `result`           | varchar         | 100       |                                                                 |         |
-| `status`           | varchar         | 100       |                                                                 |         |
-| `type`             | varchar         | 100       | to indicate this is CI or CD                                    |         |
-| `duration_sec`     | bigint unsigned |            |                                                                 |         |
-| `created_date`     | datetime        | 3         |                                                                 |         |
-| `finished_date`    | datetime        | 3         |                                                                 |         |
-
-#### cicd_pipeline_repos
-
-A map between cic_pipeline and repo info.
-
-| **field**    | **type** | **length** | **description**                                                 | **key** |
-| :----------- | :------- | :--------- | :-------------------------------------------------------------- | :------ |
-| `commit_sha` | varchar  | 255       |                                                                 | PK      |
-| `branch`     | varchar  | 255       |                                                                 |         |
-| `repo_url`   | varchar  | 255       |                                                                 |         |
-| `id`         | varchar  | 255       | This key is generated based on details from the original plugin | PK      |
-| `created_at` | datetime | 3         |                                                                 |         |
-| `updated_at` | datetime | 3         |                                                                 |         |
-
+| **field** | **type** | **length** | **description**                                                 | **key** |
+| :-------- | :-------- | :-------- |:----------------------------------------------------------------| :-------- |
+| `id` | varchar | 255) | This key is generated based on details from the original plugin | PK |
+| `created_at` | datetime | 3)                             | Devlake common.NoPKModel                                        |  |

```

Comment:
why is there a `) ` after every length, such as `3)`, `255)`? Can we remove them?

Reply:
> why is there a `) ` after every length, such as `3)`, `255)`? Can we remove them?

my bad, fixed